protocol/src/schema/FieldInfo.ts

```diff
@@ -0,0 +1,14 @@
+import { Schema } from 'airtight';
+
+export interface FieldInfo {
+    name: string;
+    type: string;
```

Comment:
More of a question of curiosity. Wonder if it's worth having here an enum for `FieldType`?

Reply:
Possibly! The drivers I've looked at so far do only provide a type code in the fieldData which I was planning on mapping to human-readable values instead (thus a string) anyway, so we could go with an enum if its useful? I didn't want to be too restrictive with it because I think each driver has a core set of types and then a bunch of their own too, so it would mean maintaining the enum.